source/docs/networking/networking-introduction/ip-configurations.rst

```diff
@@ -26,7 +26,7 @@ This section describes networking when connected to the Field Network for match
 On the Field DHCP Configuration
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The Field Network runs a DHCP server with pools for each team that will hand our addresses in the range of 10.TE.AM.20 and up with subnet masks
+The Field Network runs a DHCP server with pools for each team that will hand out addresses in the range of 10.TE.AM.20 to 10.TE.AM.199 with a subnet mask of 255.255.255.0, and a default gateway of 10.TE.AM.4.
```

Comment:
While we are here we probably should clarify that the radio has its own DHCP server that passes out IPs to the robot local network. And the field only passes IPs to the local field network. No DHCP traffic should flow over the wireless link.

Reply:
Agreed.  That would help add additional context.